### ISSUE TITLE
PRA?

### ISSUE THREAD

**Issue by dluetger** [18F/usfs-special-uses-prototype#27]

**Comment by neilmb** [comment]
We should make contact with the Privacy office within Forest Service to talk about this. Maybe it should wait until we get a product owner on board, but maybe we could just talk about it hypothetically and on background with them now.

**Comment by neilmb** [comment]
Sent an email to Brian Schwind to get the name of a contact.

**Comment by neilmb** [comment]
Meeting is set up for 8/26. I'll drop this "In Review" until that meeting happens.

**Comment by christophermather** [comment]
@neilmb - when you have a chance, can you summarize what we learned in yesterday's meeting here?

Once that's done, two questions:

- Is this issue ready to close?
- Is there a follow-on task that we should add to our backlog?

**Comment by neilmb** [comment]
Summarized information we learned here and created #73 for follow-on work.
